pkgs/development/python-modules/flask-basicauth/default.nix

```
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, flask
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "Flask-BasicAuth";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jpvanhal";
+    repo = pname;
+    rev = "632db23f547d727be378eb636187847100e2992b";
+    sha256 = "sha256-p9/IfnsOlL8EMxqQ/8J7VdJa8bMpkDHaX2ZLE3k/uFw=";
+  };
+
+  propagatedBuildInputs = [ flask ];
+
+  patches = [
+    (fetchpatch {
+      # changes imports in tests from flask.ext.basicauth to flask_basicauth
```

You should add a name to the fetched patch, this also serves the purpose of the comment. (Feel free to change the name if you'd like, but I think it's fine.)

```suggestion
      name = "fix-test-imports.patch";
```

---

nice, i didn't know / forgot that `fetchpatch` had a `name`. updated. thanks!